packages/venia-ui/lib/components/ToastContainer/toastContainer.js

```diff
@@ -29,6 +42,16 @@ const ToastContainer = props => {
     );
 };
 
+/**
+ * Props for {@link ToastContainer}
+ *
+ * @typedef props
+ *
+ * @property {Object} classes An object containing the class names for the
+ *   ToastContainer and its Toast components
+ * @property {String} classes.root CSS classes for the root container
+ * @property {String} classes.item A currently unused CSS class property
```

Comment:
If you can't find a reference to this property, in any of the related toast css files, we should just delete the prop from `propTypes`.

Reply:
Prop deleted